verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -441,11 +442,14 @@ export function SortSelect({
 }: SortSelectProps): JSX.Element {
     const classes = sortSelectStyles();
 
+    const filterBreadCrumbs = useSelector(selectFilterBreadcrumbs);
+    let filteredKeys = filterBreadCrumbs.map(({ key }) => key);  
+
     const sortKeywords = [
         { name: 'None', value: SortBy.Default },
         // Commenting out until fix is found for big queries sorting in mongodb
         // { name: 'Confirmed date', value: SortBy.ConfirmedDate },
-        { name: 'Country', value: SortBy.Country },
+        {...filteredKeys[0] !== 'country' && { name: 'Country', value: SortBy.Country }},
```

Comment:
You should use `filteredKeys.includes('country')` here, because country filter won't always be first